addons/repair/models/stock_lot.py

```diff
@@ -19,9 +20,13 @@ def _compute_repair_order_ids(self):
             lot.repair_order_ids = repair_orders[lot.id]
             lot.repair_order_count = len(lot.repair_order_ids)
 
+    def _compute_in_repair_count(self):
+        for lot in self:
+            lot.in_repair_count = len(self.env['repair.order'].search([('lot_id', '=', lot.id), ('state', 'not in', ('done', 'cancel'))]))
```

Comment:
Why not use search_count to optimize the query ?
```suggestion
            lot.in_repair_count = self.env['repair.order'].search_count([('lot_id', '=', lot.id), ('state', 'not in', ('done', 'cancel'))]))
```

Reply:
done